### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by antebrl** [Pullolo/DiamondCasino#5]
A few Ideas I have. Would be cool to see some being implemented!
- [ ] Also enable betting with other items or XP as currency
- [ ] Chat message if somebody wins (big - maybe set the amount in config)
- [ ] Multi players at one blackjack table, seeing each others bets (saw this in a other casino plugin)
- [ ] More animations
- [ ] More Games (slot, ..)
- [ ] Dice game favors the player at first glance? Shouldn't do that, or make the odds configurable!

**Comment by RabuSoore** [comment]
Yes, I agree, also add a token system, you can customize the price per token in the config.
